test/fail/verify.sh

```diff
@@ -29,4 +29,4 @@ fi
 
 set -e
 
-test ! -e /test.css
+test ! -e test.css
```

Comment:
This was missed in the previous refactor - would have always "passed" as `test.css` would never exist at root path.

Reply:
haha wooops!